Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -322,22 +310,123 @@ public void Exit()
             _exitTriggered = true;
         }
 
+        private Order GetOrderById(int id)
+        {
+            Order order = null;
+
+            if (_orders.TryGetValue(id, out order) == false)
+            {
+                var clientOrder = _algorithm.Transactions.GetOrderById(id);
+
+                if (order != null)
```

Comment:
https://github.com/QuantConnect/Lean/pull/134/files#diff-93fd550d3a43e7fb3ded36524aa7aefbL257

Because of changes ot BrokerageSetupHandler, now it's possible for Transactions to have an order that BrokerageTransactionHandler doesn't